### ISSUE TITLE
Error on OS X with .DS_Store

### ISSUE THREAD

**Issue by hypery2k** [alexec/docker-maven-plugin#44]
I rarely get errors with .DS_Store files
```
Failed to execute goal com.alexecollins.docker:docker-maven-plugin:2.4.0:start (docker-prepare) on project webtest: java.io.IOException: Source '../src/main/docker/.DS_Store' exists but is not a directory -
```
Would be nice to filter them out by the plugin.

BTW: Awesome work. I love this plugin. Thanks for the great work

**Comment by alexec** [comment]
Ok. I will not get a chance to sort this out. You could get the maven clean plugin to delete them as a workaround?

**Comment by alexec** [comment]
Fixed in 2.4.1.

**Comment by hypery2k** [comment]
thanks
